web-console/src/lib/components/pipelines/editor/TabLogs.svelte

```diff
@@ -51,7 +51,7 @@
       // Close log stream when leaving log tab, or switching to another pipeline
       let oldPipelineName = pipelineName
       return () => {
-        if ('open' in streams[oldPipelineName].stream) {
+        if (streams[oldPipelineName] && 'open' in streams[oldPipelineName].stream) {
```

Comment:
Did it before throw a key error here?

Reply:
The situation may occur where the value for "oldPipeline" key has already been deleted